dashboard/cron.py

```diff
@@ -128,25 +128,29 @@ def update_user(self):
         # loop through multiple course ids
         for data_warehouse_course_id in Course.objects.get_supported_courses():
 
-            # select all student registered for the course
-            user_sql = f"""select u.name AS name,
-                        p.sis_user_id AS sis_id,
-                        p.unique_name AS sis_name,
-                        u.global_canvas_id AS id,
-                        c.current_score AS current_grade,
-                        c.final_score AS final_grade,
-                        '{data_warehouse_course_id}' as course_id
-                        from user_dim u,
-                        pseudonym_dim p,
-                        course_score_fact c,
-                        (select e.user_id as user_id, e.id as enrollment_id from enrollment_dim e
-                        where e.course_id = '{data_warehouse_course_id}'
-                        and e.type='StudentEnrollment'
-                        and e.workflow_state='active' ) as e
-                        where p.user_id=u.id
-                        and u.id = e.user_id
-                        and c.enrollment_id =  e.enrollment_id
-                        and p.sis_user_id is not null
+            # select all student, tas, and instructors registered for the course
+            user_sql = f"""
```

Comment:
There is something wrong with this query, I see that if you just have single course the loading the data through cron process works fine, but if you have multiple courses i see this error.
```sqlalchemy.exc.IntegrityError: (_mysql_exceptions.IntegrityError) (1062, "Duplicate entry '17700000000123456-17700000000098765' for key 'PRIMARY'") (Background on this error at: http://sqlalche.me/e/gkpj)```